app_spec/test/index.js

```diff
@@ -68,16 +68,9 @@ const registerAllScenarios = () => {
   // test cases if there is any Promise awaiting in between test declarations.
   let numRegistered = 0
 
-  const registerer = orchestrator => {
-    const f = (...info) => {
-      numRegistered += 1
-      return orchestrator.registerScenario(...info)
-    }
-    f.only = (...info) => {
-      numRegistered += 1
-      return orchestrator.registerScenario.only(...info)
-    }
-    return f
+  const registerer = orchestrator => (...info) => {
```

Comment:
It looks like a merge mistake here removing the .only

Reply:
yeah, fixed.